src/storage/engine/raftkv.rs

```diff
@@ -283,13 +295,16 @@ impl<S: RaftStoreRouter> Engine for RaftKv<S> {
     fn async_write(
         &self,
         ctx: &Context,
-        mut modifies: Vec<Modify>,
+        modifies: Vec<Modify>,
         cb: Callback<()>,
     ) -> engine::Result<()> {
         fail_point!("raftkv_async_write");
+        if modifies.is_empty() {
```

Comment:
Any tests to cover it?

Reply:
Yes, https://github.com/pingcap/tikv/pull/2625/files#diff-6558c014328a8c95a1da6a12faad85e3L304

Reply:
It tests against local engine instead of raftkv.

Reply:
It is testd on raftkv. https://github.com/pingcap/tikv/pull/2625/files/a550b721f0187168da4b54fedc9be832bf4f0da9#diff-6558c014328a8c95a1da6a12faad85e3L20